clang/lib/Driver/ToolChains/CommonArgs.cpp

```diff
@@ -1441,8 +1441,10 @@ collectSanitizerRuntimes(const ToolChain &TC, const ArgList &Args,
     if (SanArgs.needsScudoRt()) {
       SharedRuntimes.push_back("scudo_standalone");
     }
-    if (SanArgs.needsTsanRt())
+    if (SanArgs.needsTsanRt() && SanArgs.linkRuntimes())
```

Comment:
How is this change to tsan related to tysan?

Reply:
Nope, I think this was left over from a merge, removed, thanks!